CollectSamError/CollectSamErrorMetrics.wdl

```diff
@@ -43,15 +43,23 @@ task CollectSamErrorMetricsTask {
         File intervals
         Int disk_gb = 400
         Int memory_gb = 16
+        Int minMQ = 60
+
     }
     command {
         java -Xms2000m -jar /usr/gitc/picard.jar CollectSamErrorMetrics I=~{input_bam} \
-        O=~{output_name} R=~{ref_fasta} V=~{vcf_file} L=~{intervals} \
+        O=~{output_name} R=~{ref_fasta} V=~{vcf_file} L=~{intervals} MQ=~{minMQ} \
         ERROR_METRICS=null \
         ERROR_METRICS="ERROR:INSERT_LENGTH" \
```

Comment:
Can you add:
OVERLAPPING_ERROR:INSERT_LENGTH
OVERLAPPING_ERROR:GC_CONTENT

and add an option that we can expose to Terra to the probability with a default of 1.